_posts/2024-04-02-how-to-identify-dropped-leads-using-7ts-chrome-extension.md

```diff
@@ -0,0 +1,68 @@
+---
+date: 2024-02-04
+title: How to Identify Dropped Leads Using the 7Targets Chrome Extension
+#video_id: JC2yJnBXzLM
+description: Learn how a user identifies dropped leads in their Gmail sent box.
+type: How-to
+sidebar:
+  - {id: overview, text: Overview}
+  - {id: Installing the Extension, text: Installing the Extension}
+  - {id: Gmail Authorization, text: Gmail Authorization}
+  - {id: Dropped Lead Identification, text: Dropped Lead Identification}
+  - {id: Managing Dropped Leads and Reactivations, text: Managing Dropped Leads and Reactivations}
+
+categories:
+  - How-To
+# set: getting-started
+# set_order: 3
+---
+
+## Overview
+Learn to efficiently identify dropped leads using the 7Targets Chrome Extension within Gmail. Discover how to spot potential customers who've lost interest and reactivate them. This tutorial guides you through authorizing the extension, identifying dropped leads, and reviving them for increased sales. Unlock the power of the 7Targets dropped leads Extension to boost your lead management and business growth.
+
+## Installing the Extension
+To get started with the 7Targets Chrome Extension, follow these simple steps:
+
+1. Visit the Chrome Web Store and search for "7Targets Dropped Leads Extension".

```

Comment:
Lets also provide a link to the zip till the time we are not on the webstore. 

1. Visit the Chrome Web Store and search for "7Targets Dropped Leads Extension" or Download it from <here>

Reply:
done